input/postgres/helpers.go

```diff
@@ -72,16 +71,10 @@ SELECT setting
 	FROM pg_settings
  WHERE name = '%s'`
 
-func GetPostgresSetting(ctx context.Context, settingName string, server *state.Server, globalCollectionOpts state.CollectionOpts, prefixedLogger *util.Logger) (string, error) {
+func GetPostgresSetting(ctx context.Context, db *sql.DB, settingName string) (string, error) {
```

Comment:
This looks cleaner, but I wonder why we weren't doing this previously 🤔

Reply:
Looks like it started out being used in a context where we didn't have a pre-existing connection (`runner/logs.go` in https://github.com/pganalyze/collector/commit/cbf2f087d6ee12fc3b3eefdd88ebd6402d97264c), and we later used the same helper in other places, without refactoring it.